site/content/docs/5.3/utilities/object-fit.md

```diff
@@ -25,23 +25,23 @@ Classes for the value of `object-fit` are named using the format `.object-fit-{v
-{{< placeholder width="140" height="120" class="object-fit-cover border" text="Object fit cover" markup="img" >}}
-{{< placeholder width="140" height="120" class="object-fit-fill border" text="Object fit fill" markup="img" >}}
-{{< placeholder width="140" height="120" class="object-fit-scale border" text="Object fit scale down" markup="img" >}}
-{{< placeholder width="140" height="120" class="object-fit-none border" text="Object fit none" markup="img" >}}
+{{< placeholder width="140" height="120" class="object-fit-contain border" text="Object fit contain" markup="img" color="#868e96" background="#dee2e6" >}}
+{{< placeholder width="140" height="120" class="object-fit-cover border" text="Object fit cover" markup="img" color="#868e96" background="#dee2e6" >}}
+{{< placeholder width="140" height="120" class="object-fit-fill border" text="Object fit fill" markup="img" color="#868e96" background="#dee2e6" >}}
+{{< placeholder width="140" height="120" class="object-fit-scale border" text="Object fit scale down" markup="img" color="#868e96" background="#dee2e6" >}}
+{{< placeholder width="140" height="120" class="object-fit-none border" text="Object fit none" markup="img" color="#868e96" background="#dee2e6" >}}
 {{< /example >}}
 
 ## Responsive
 
 Responsive variations also exist for each `object-fit` value using the format `.object-fit-{breakpoint}-{value}`, for the following breakpoint abbreviations: `sm`, `md`, `lg`, `xl`, and `xxl`. Classes can be combined for various effects as you need.
 
 {{< example class="d-flex overflow-auto" >}}
-{{< placeholder width="140" height="80" class="object-fit-sm-contain border" text="Contain on sm" markup="img" >}}
-{{< placeholder width="140" height="80" class="object-fit-md-contain border" text="Contain on md" markup="img" >}}
-{{< placeholder width="140" height="80" class="object-fit-lg-contain border" text="Contain on lg" markup="img" >}}
-{{< placeholder width="140" height="80" class="object-fit-xl-contain border" text="Contain on xl" markup="img" >}}
-{{< placeholder width="140" height="80" class="object-fit-xxl-contain border" text="Contain on xxl" markup="img" >}}
+{{< placeholder width="140" height="80" class="object-fit-sm-contain border" text="Contain on sm" markup="img" color="#868e96" background="#dee2e6" >}}
+{{< placeholder width="140" height="80" class="object-fit-md-contain border" text="Contain on md" markup="img" color="#868e96" background="#dee2e6" >}}
+{{< placeholder width="140" height="80" class="object-fit-lg-contain border" text="Contain on lg" markup="img" color="#868e96" background="#dee2e6" >}}
+{{< placeholder width="140" height="80" class="object-fit-xl-contain border" text="Contain on xl" markup="img" color="#868e96" background="#dee2e6" >}}
+{{< placeholder width="140" height="80" class="object-fit-xxl-contain border" text="Contain on xxl" markup="img" color="#868e96" background="#dee2e6" >}}
```

Comment:
Should we try to use Orange color in here ? For instance, `#999999` or `#666666` and `#eeeeee` or `#dddddd`

Reply:
Changed it via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1907/commits/3e7653e0d20b810bcd0c36b62be9b2ac6c69a37a